File: core/core-api/src/main/java/com/nmh/commerce/controller/HealthController.java
package com.nmh.commerce.controller;

import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HealthController {

    public ResponseEntity<Object> health() {
        return ResponseEntity.status(HttpStatus.OK).build();
    }

}

File: core/core-api/src/main/kotlin/com/nmh/commerce/controller/HealthController.kt
package com.nmh.commerce.controller

import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthController {
    fun health(): ResponseEntity<Any?> = ResponseEntity.status(HttpStatus.OK).build<Any?>()
}
